monocle/webapp.py

```diff
@@ -30,14 +31,15 @@
 
 
 app = Flask(__name__)
-CORS(app)
+CORS(app, resources={r"/api/0/query/*":
+                     {"origins": os.getenv('ALLOW_ORIGIN', '*')}})
 
 
 @app.route("/api/0/query/<name>", methods=['GET'])
 def query(name):
     repository_fullname = request.args.get('repository')
     params = utils.set_params(request.args)
-    db = ELmonocleDB()
+    db = ELmonocleDB(os.getenv('ELASTIC_CONN', 'localhost:9200'))
```

Comment:
As the api will be nested in the monocle cli, we could remove the env vars input as well.

Reply:
But again can be fixed wth a followup pr.